dev/BackgroundTask/BackgroundTaskBuilder/BackgroundTaskBuilder.idl

```diff
@@ -24,6 +24,7 @@ namespace Microsoft.Windows.ApplicationModel.Background
             );
 
         String Name{ set; get; };
+        Windows.ApplicationModel.Background.BackgroundTaskRegistrationGroup TaskGroup{ set; get; };
```

Comment:
Are we going to expose other properties as well from UWP's BackgroundTaskBuilder? Like we have exposed AddCondition method but do not allow modifications to CancelOnConditionLoss property. What's the plan for other remaining properties.

Reply:
CancelOnConditionLoss will not help as the win32 process won't be killed if condition is lost. If required, apps can have listeners and close win32 process if required. Other properties are also not meaningful for COM BG Task.